dvo_extractor/command_line.py

```diff
@@ -38,7 +38,7 @@ def print_version():
     )
 
 
-def insights_sha_extractor():
+def insights_sha_extractor() -> None:
```

Comment:
shouldn't this function be dvo, not sha?

Reply:
you guess ;)

Reply:
fixed, TY